main.html

```diff
@@ -0,0 +1,237 @@
+            </div>
+        </div>
+        
+        <!--2-->
+        <div class="row"> 
+            <div class="col-md-6 mt-5">
+                <img class="im" src="../img/IMG_0089.JPG" alt="">    
+            </div>
+
+            <div class="col-md-6 mt-5">
+                <img class="im" src="../img/IMG_0492.JPG" alt=""> 
+            </div> 
+        </div>
+
+         <!--3-->
+         <div class="row">
+            <div class="col-md-4 mt-5">
+                <img class="im" src="./img/IMG_0096.JPG" alt="">    
+            </div>
+    
+            <div class="col-md-4 mt-5">
+                <img class="im" src="./img/IMG_0509.JPG" alt=""> 
+            </div>
+    
+            <div class="col-md-4 mt-5">
+                <img class="im" src="./img/IMG_0103.JPG" alt="">    
+            </div>
+        </div>
+        
+        <!--2-->
+        <div class="row">
+            <div class="col-md-6 mt-5">
+                <img class="im" src="../img/IMG_0276.JPG" alt=""> 
+            </div>  
+            <div class="col-md-6 mt-5">
+                <img class="im" src="../img/IMG_0453.JPG" alt=""> 
+            </div> 
+        </div>
+        <!--solo-->
+        <div class="row justify-content-center">
+            <div class="col mt-5 mb-5">
+                <img class="im" src="../img/IMG_1814.JPG" alt=""> 
+            </div> 
+        </div>
+    </div>
+
+    <!--Bio-->
+    <h1 class="bheading mb-5 text-center">Bio</h1>
+    <div  id="bio" class=" mt-5 mb-5 container"> 
+        <div class="bb row justify-content-center">
+            <div class="col-md-3">
+                <img class="w-100" src="./img/chris_Maskot copy.png" alt=""> 
+            </div>
+            <div class="col-md-9 align-self-center">
+                <p style="background-color: #000;"class=" p-5 lead text-light text-left"> Motivation, Ambition, determination, and faith, are all Ct Native Need in order to excel. 
+                    Hustle-minded, fast paced industry. Christopher Micheal was born in Danbury CT and raised in Norwalk CT on August 28, 1999. 
+                    Growing up, he had many influences to help drive him into being self Employed. His influences were Steve Jobs, Elon Musk, Mark Zuckerberg. 
+                    Christopher began developoing websites at the age of 16, Creating and redesigning websites he saw in high School. He also took a class for Web Design his junior year of High School. 
+                    At that moment he found his desire to become a professional. He has created 4 actual sites for clients so far. Example of past clients are, Construction Company, Musical Artist, Dj and a Car Mechanic. 
+                    Christopher Micheal is still on his grind to the top, and absolutely nothing stopping him From keeping his ambition.</p>
+            </div>
+        </div>
+    </div>
+
+    <!--footer-->
+    <div class=" footer container-expand mb-2">
```

Comment:
Consider adding some more information down here, like Copyright (you can use JavaScript to always show the current year). You could also have an email link down here (which would be in addition to the form I suggested in your page).